<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar</class>
 <widget class="QDialog" name="Calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>631</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(103, 96, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QCalendarWidget" name="calendar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>461</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						alternate-background-color: rgb(255, 170, 127);
						color: rgb(6, 6, 6);
						background-color: rgb(255, 239, 231);
						font-size: 10pt;
						font-weight:600;
					</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <widget class="QWidget" name="Kalendarz" native="true">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>-10</y>
      <width>431</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 145, 234);</string>
    </property>
    <widget class="QPushButton" name="SaveButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>440</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							background-color: rgb(255, 214, 180);
							font-size:12pt;
							font-weight:550;
						</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="Napis2_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							color: rgb(255, 255, 255);
							font-size:16pt;
						</string>
     </property>
     <property name="text">
      <string>your activity details</string>
     </property>
    </widget>
    <widget class="QLabel" name="Napis2_1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							font-size:18pt;
							font-weight:600;
							color: rgb(255, 255, 255);
						</string>
     </property>
     <property name="text">
      <string>Provide</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>91</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							font-size:13pt;
							color: rgb(255, 255, 255);
						</string>
     </property>
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>310</y>
       <width>61</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							font-size:15pt;
							color: rgb(255, 255, 255);
						</string>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Napis1_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						font-size:18pt;
						font-weight:600;
						color: rgb(255, 255, 255);
					</string>
    </property>
    <property name="text">
     <string>Monthly calendar </string>
    </property>
   </widget>
   <widget class="QLabel" name="Napis1_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						color: rgb(255, 255, 255);
						font-size:16pt;
					</string>
    </property>
    <property name="text">
     <string>for running</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
